src/components/IncidentDetails/index.js

```diff
@@ -0,0 +1,181 @@
+import React, { useMemo } from "react";
+import { useForm } from "react-hook-form";
+import dayjs from "dayjs";
+import { IncidentDetailsRow } from "./IncidentDetailsRow";
+import { Select } from "../Select";
+import { IncidentPriority } from "../../constants/incident-priority";
+import {
+  IncidentCommandersOption,
+  IncidentCommandersSingleValue,
+  IncidentPriorityOption,
+  IncidentPrioritySingleValue,
+  IncidentRespondentsMultiValueLabel,
+  IncidentRespondentsOption
+} from "./select-components";
+import { personRespondents, priorities } from "./data";
+
+export const IncidentDetails = ({
+  incident,
+  node,
+  updateStatusHandler,
+  textButton
+}) => {
+  // Temporary mock, in the future you need to replace it with an array of real users received from the api
+  const commandersArray = useMemo(
+    () => [
+      {
+        label: incident.commander_id.name,
+        value: incident.commander_id.id,
+        avatar: incident.commander_id.avatar
+      }
+    ],
+    [incident]
+  );
+
+  const { control, getValues, watch } = useForm({
+    defaultValues: {
+      tracking: "123456",
+      started: incident.created_at,
+      duration: incident.created_at,
```

Comment:
```suggestion
      created_at: incident.created_at,
```

здесь должны быть данные инцидента. что там с них ты уже отрендеришь - другой вопрос. хоть и 10 надписей с одной created_at даты